monitoring/README.md

```diff
@@ -7,4 +7,13 @@ docker-compose up -d grafana prometheus
 ./grafana/bin/upload-dashboards.sh
 ```
 
-Visit `http://${REMOTE_DEV_HOST}:3000`.
\ No newline at end of file
+Visit `http://${REMOTE_DEV_HOST}:3000`.
+
+### Deploy Changes
+
+```bash
+ssh prometheus-grafana-metrics
+cd ~/audius-protocol/monitoring
+git checkout master
```

Comment:
should do a `git pull origin master` here?

Reply:
Thanks!

The `git pull` is already happening with the deploy script: https://github.com/AudiusProject/audius-protocol/pull/3228/files#diff-ae5a05e31eca7c4be5060db4c336f48c4de53189fc8e71c4358fd861f957161fR5.

That way it always gets changes, no matter what branch I'm deploying from.